hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -666,15 +667,18 @@ public int getAutoAssociationsCount(
             return 0;
         }
         for (var transfer : cryptoTransferBody.tokenTransfers()) {
-            final var tokenID = transfer.token();
             autoAssociationsCount += (int) transfer.transfers().stream()
                     .filter(accountAmount -> accountAmount.amount() > 0)
-                    .map(AccountAmount::accountID)
-                    .filter(accountID -> hasNoRelation(relationStore, accountID, tokenID))
+                    // FUTURE: This is a temporary performance fix to avoid state reads for auto-associations
```

Comment:
I think this makes the throttles more severe instead of disabling them because it accounts for tokens that already are associated. Maybe it is sufficient to return 0 from this method.